src/ec/packages/ec-component-description-list/description-list.html.twig

```diff
@@ -23,10 +24,15 @@
 
 {% set _items = items|default([]) %}
 {% set _css_class = 'ecl-description-list' %}
+{% set _variant = variant|default('') %}
 {% set _extra_attributes = '' %}
 
 {# Internal logic - Process properties #}
 
+{% if (_variant is not empty) %}
```

Comment:
parentheses around the condition really needed?

Reply:
not at all.. :)